src/main/kotlin/com/exactpro/th2/common/schema/message/DeliveryMetadata.kt

```diff
@@ -17,5 +17,6 @@ package com.exactpro.th2.common.schema.message
 
 data class DeliveryMetadata(
     val consumerTag: String,
+    val deliveryTag: Long,
```

Comment:
Why do we need a delivery tag here? It is a rabbitmq specific but we expose it as a public API... The only place you use it - tests. Probably, the content itself should be enough for tests...